numpy/core/tests/examples/cython/meson.build

```diff
@@ -0,0 +1,27 @@
+project('checks', 'c', 'cython')
+
+py_mod = import('python')
+py3 = py_mod.find_installation(pure: false)
+
+cc = meson.get_compiler('c')
+cy = meson.get_compiler('cython')
+
+if not cy.version().version_compare('>=0.29.35')
+  error('tests requires Cython >= 0.29.35')
+endif
+
+npy_include_path = run_command(py3, ['-c',
+               'import os; os.chdir(".."); import numpy; print(os.path.abspath(numpy.get_include()))'],
```

Comment:
indentation is quite messy here. if you move `['-c'` to this line and indent it by 4 spaces, it'll look better.

Reply:
(comments on this file apply to the other one too)